modules/appeals_api/spec/requests/v2/notice_of_disagreements_controller_spec.rb

```diff
@@ -200,53 +193,19 @@ def new_base_path(path)
       let(:path) { base_path('notice_of_disagreements') }
 
       it 'evidence_submission' do
-        post(path, params: @minimum_data, headers: @headers)
+        post(path, params: minimum_data, headers:)
         nod = AppealsApi::NoticeOfDisagreement.find_by(id: parsed['data']['id'])
 
         expect(nod.board_review_option).to eq('evidence_submission')
       end
 
       it 'hearing' do
-        post(path, params: @max_data, headers: @max_headers)
+        post(path, params: max_data, headers: max_headers)
         nod = AppealsApi::NoticeOfDisagreement.find_by(id: parsed['data']['id'])
 
         expect(nod.board_review_option).to eq('hearing')
       end
     end
-
```

Comment:
I took this opportunity to relocate these older oauth specs:

Initially, when the controllers for the v0 APIs were first created, the specs for the oauth versions of the routes were included here, in the same file as the decision reviews v2 specs. To clean things up, I've moved them out of this file and into the v0-specific specs, adding new files and test cases where needed.